### DISCUSSION TITLE
The API interface returns a time format problem

### DISCUSSION THREAD

**Discussion by fzwu** [goharbor/harbor#18539] [Q&A]
API  /api/v2.0/projects/{param.project_name}/repositories/{param.repository_name}/artifacts
![image](https://user-images.githubusercontent.com/16144784/232370306-dbf4da4b-4598-4267-b01a-d80f6e2c6447.png)

![image](https://user-images.githubusercontent.com/16144784/232370252-76e9f274-3aa2-4367-b94b-2d920c17dd3d.png)

The difference between the time displayed on the UI interface and the time returned by the API interface is 8 hours. How can I set the difference

**Comment by stonezdj** [comment]
The API time is UTC time and the UI displays the time in current TimeZone

**Reply by fzwu** [reply]
How do I change the API time to current TimeZone？Or which configuration item can support changing the API time
